jobs/live-better-with-lead-developer.html

```diff
@@ -1,13 +1,13 @@
 ---
-title: Junior and Mid-weight Python Developers
+title: Lead Developer
 company: Live Better With
 location: London, England
```

Comment:
You can give a more precise address - you could rewrite this line as "Kings Cross, London, UK"